tasks/main.yml

```diff
@@ -89,19 +89,27 @@
   import_tasks: virtualenv.yml
   when: ansible_os_family in ['RedHat', 'Darwin']
 
+- name: define private key path
```

Comment:
This can be removed after my comment on the `generate the private key` task.

Reply:
Implemented as you've suggested.